responders/src/UI/embc-responder/src/app/feature-components/wizard/support-components/select-support/select-support.component.html

```diff
@@ -71,52 +71,48 @@
             </div>
           </div>
 
-          <div
-            class="row"
-            *ngIf="
-              !evacueeSessionService?.currentNeedsAssessment?.canProvideFood
-            "
-          >
-            <div class="col-4">Food</div>
+          <div *ngIf="doesIncludeNeed(null)" class="row">
```

Comment:
probably deserves its own function? hasNoNeeds()

Reply:
I can if you want but I'm not sure that's necessary, it's checking the condition that there is nothing in the IdentifiedNeeds array to display the text if that's true, but I can make a new function just for that if you like

Reply:
my preference is to move these functions to the code behind instead of embedding in the html